content/en/apps/guides/performance/telemetry.md

```diff
@@ -97,4 +97,15 @@ When the aggregate doc is created the Telemetry service also includes a snapshot
 | `dbInfo.idb_attachment_format` | The format of database attachments. |
 | `dbInfo.db_name` | The name of the local database. |
 | `dbInfo.auto_compaction` | Whether or not auto compaction is set. |
-| `dbInfo.adapter` | The database adapter being used. |
\ No newline at end of file
+| `dbInfo.adapter` | The database adapter being used. |
+
+## Export data to JSON
+
+While you can view the telemetry in the database with a tool like [Fauxton](https://couchdb.apache.org/fauxton-visual-guide/), you may also want to export the data to JSON to further analyze or visualize it.  To do so:
```

Comment:
I'd either remove the fauxton bit, or guide folks to how/where to see telemetry data in fauxton.

```suggestion
{{% alert title="Note" %}}
Telemetry data can be viewed directly in your browser with [Fauxton](https://couchdb.apache.org/fauxton-visual-guide/) at `https://{{CHT_INSTANCE_URL}}/_utils`, and navigating the `medic-users-meta` database.
{{% /alert %}}

Follow the steps below to export all telemetry in JSON for further analysis or visualization:
```

Reply:
Ah - the "note" is a nice touch - thanks! Fixed in 23e949b